tests/director/generate_examples.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
I do not get the magic that happens here :-(

Reply:
As the tests are ran into a pod, the generated examples are saved onto the pod file system and when the pod is destroyed the new examples are lost. We mount our local /examples to /examples into the pod using minikube mount in order for the examples to be stored locally.  The mount process needs to be kept alive during the tests execution and killed when the tests finish. That is why we use the trap. Then we just create clusterTestSuite that maches the director's testDefinition and the tests are executed. The rest of the script just gives us updates on the suite's status.